static/src/stylesheets/layout/nav/_top-bar-dropdown.scss

```diff
@@ -32,7 +50,35 @@
         border-top: 1px solid $brightness-93;
         display: block;
         top: 0;
-        margin: -1px 0 0 $gs-gutter * 1.5;
+        margin: -1px 0 0 $gs-gutter;
+    }
+
+    @include mq($until: tablet) {
+        position: fixed;
+        border-radius: 0;
+        top: 36px;
+        left: 0;
+        right: 0;
+        width: auto;
+
+        @media (max-height: 370px) {
```

Comment:
A comment here would be nice explaining the media query

Reply:
Is the mq necessary? Would `min-height: calc(100%-36px)` (maybe `min-height:90%` for older browsers too) + always overflowing to auto) work? That way we don't have `370px` set in stone and if the menu grows that's okay